### ISSUE TITLE
add `time` to performance related commands

### ISSUE THREAD

**Issue by rdallman** [fnproject/tutorials#5]
this was the one brought forth: https://github.com/fnproject/tutorials/blob/master/JavaFDKIntroduction/README.md#improving-performance

makes it clear how much faster things are

**Comment by michael-w-williams** [comment]
Good idea. Just need to figure out where to put that. Probably needs to be in each tutorial.
